zhaquirks/xiaomi/WXKG01LM_mija_button.py

```diff
@@ -0,0 +1,116 @@
+import logging
+import homeassistant.components.zha.const as zha_const
+from zigpy.profiles import PROFILES, zha
+from zigpy.zcl.clusters.general import LevelControl, Ota, Basic, Groups, OnOff,\
+     Identify, Scenes
+from zhaquirks.xiaomi import BasicCluster, PowerConfigurationCluster,\
+     XiaomiCustomDevice
+from zhaquirks import CustomCluster
+
+XIAOMI_CLUSTER_ID = 0xFFFF
+
+_LOGGER = logging.getLogger(__name__)
+
+click_type_map = {
+ 2: 'double',
+ 3: 'triple',
+ 4: 'quadruple',
+ 128: 'furious',
+}
+
+class MijaButton(XiaomiCustomDevice):
+
+    def __init__(self, *args, **kwargs):
+        self.battery_size = 9
+        super().__init__(*args, **kwargs)
+
+    class MijaOnOff(OnOff, CustomCluster):
+        cluster_id = OnOff.cluster_id
+
+        def __init__(self, *args, **kwargs):
+            self._currentState = {}
+            super().__init__(*args, **kwargs)
+
+        def _update_attribute(self, attrid, value):
+            _LOGGER.debug("MijaOnOff _update_attribute: %s = %s",attrid, value)
+            click_type = False
+
+            # Handle Mija OnOff
+            if(attrid == 0):
+                value = False if value else True
```

Comment:
can this be collapsed into the line below?

Reply:
I'm not sure what you mean. Set "value" and "click_type" in one line?

Reply:
nvm this